README.md

```diff
@@ -101,9 +101,15 @@ protocols:
       # Default: 0.
     protocol:
       # List of commands that your script is expected to execute.
-      - command: string
-          # the executable, followed by its arguments, separated by spaces.
+      - command|regex: string
```

Comment:
i wasn't sure how to format the idea of "either" since we have a sort of pseudo syntax here, thoughts?